comfy_cli/workflow_ops.py

```
@@ -0,0 +1,1112 @@
+        if ref.lstrip("-").isdigit():
+            return int(ref)
+    return ref
+
+
+def _split_ref_slot(spec_val: str, aliases: dict[str, Any]) -> tuple[Any, Any]:
+    """Split `<node_or_alias>.<slot>` and resolve the node part."""
+    node_part, _, slot = str(spec_val).partition(".")
+    return resolve_ref(node_part, aliases), slot
+
+
+def apply_specs(workflow: dict, graph, specs: list, *, actor: str = "cli", base_version: int = 0) -> tuple[dict, list, dict]:
```

Pylint (4.0.6)

[convention] Line too long (125/120)

(C0301)